e2e-tests/conf/minio-secret.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
In other operators we call this file `backup-s3.yaml` and the secret `my-cluster-name-backup-s3`. Consider having it the same.

Reply:
sorry, wrong file, I meant about s3credentials.yaml, but I'll create a PR later for that.